generic/ltp.py

```diff
@@ -126,11 +133,11 @@ def setUp(self):
     def test(self):
         logfile = os.path.join(self.logdir, 'ltp.log')
         failcmdfile = os.path.join(self.logdir, 'failcmdfile')
-
+        skipfile = self.params.get('skipfile', default='skipfile')
         os.chmod(self.teststmpdir, 0o755)
         self.args += (" -q -p -l %s -C %s -d %s -S %s"
                       % (logfile, failcmdfile, self.teststmpdir,
-                         self.get_data('skipfile')))
+                         self.get_data(skipfile)))
```

Comment:
This restricts the user to place the file in data directory. I would suggest it not to do that and instead it should take an absolute path

Reply:
can you suggest the path as i am looking this  for  on use case or if you say can comment in yaml file as it need to be placed in data dir ?

Reply:
the implementation in script should be to take from data dir by default, and it can optionally be overwritten by user. you get the point?

Reply:
no i don't want that way as skipfile name can be changed in my case as want to maintain more skipfile having different set  of test to skip

Reply:
I am not sure if you follow me here. Considering only one file is going to be used, if the input path can be absolute (be it data directory or any other directory) you can still have different skipfiles as well right? But the existing approach restricts user to have skipfiles only in data directory.

Reply:
sorry not very clear to me as i got let me write you are saying we can code skipfile don't get from datadir if it is custom it can be absolute path from system  or http path ?

Reply:
yes, It can be absolute path from the system is what I meant.

Reply:
@harish-24 ah some issue with this approach as if i want to use this in CI i need to have independent logic to place this file in system , so i feel i can go with network path and  using asset fetch skipfile can be populated  is this  approach make sense to you ?

Reply:
@PraveenPenguin It is up-to you to enhance the test better for requirement. But make sure the existing way of providing does not break as it can be used in other places already.

Reply:
@harish-24 sure thanks for review  i will adopt that